core/commands/storage.go

```diff
@@ -618,9 +677,9 @@ the chunk and replies back to client for the next challenge step.`,
 	},
 	Arguments: []cmds.Argument{
 		cmds.StringArg("session-id", true, false, "ID for the entire storage upload session."),
-		cmds.StringArg("channel-id", true, false, "Open channel id for payment."),
 		cmds.StringArg("chunk-hash", true, false, "Chunk the storage node should fetch."),
 		cmds.StringArg("price", true, false, "Price per GB in BTT for storing this chunk offered by client."),
+		cmds.StringArg("contract", true, false, "client init contract"),
```

Comment:
Not really sure what this means, could you write a full sentence about this variable?